.github/workflows/release.yml

```diff
@@ -6,7 +6,7 @@ on:
 name: Containerd Release
 
 env:
-  GO_VERSION: "1.20.8"
+  GO_VERSION: "1.21.3"
```

Comment:
Shouldnt this be `1.20.10` ?

If we are releasing the binaries with `1.21.3`, does it make sense to run the CI with `1.20.8` also in the matrix?

Reply:
I think we should use `go1.20.10` here.